pkg/install/v1alpha1/cstor_volume.go

```diff
@@ -338,7 +359,7 @@ spec:
             openebs.io/persistent-volume: {{ .Volume.owner }}
             openebs.io/persistent-volume-claim: {{ .Volume.pvc }}
         spec:
-          serviceAccountName: {{ .Config.ServiceAccountName.value }}
+          serviceAccountName: {{ .Config.PVCServiceAccount.value | default .Config.ServiceAccountName.value }}
```

Comment:
Can we change the config name to `.Config.PVCServiceAccountName`
This will look familiar to `.Config.ServiceAccountName`